app/src/main/java/org/mozilla/fenix/tabtray/TabTrayView.kt

```diff
@@ -666,7 +689,8 @@ class TabTrayView(
         private const val TAB_COUNT_SHOW_CFR = 6
         private const val DEFAULT_TAB_ID = 0
         private const val PRIVATE_TAB_ID = 1
-        private const val EXPAND_AT_SIZE = 3
+        private const val EXPAND_AT_LIST_SIZE = 4
```

Comment:
Let's add some comments for these 2 constants too.

`
// Minimum number of grid tabs to trigger a fully expanded bottom sheet when opening the tab tray.
`

Reply:
Is it ok to use ```// Minimum number of grid tabs for which to show the tabs tray as expanded.```?

Reply:
Ya sounds good.